assets/stylesheets/assigns.scss

```diff
@@ -176,10 +176,14 @@
 
 #topic-footer-dropdown-reassign {
   .name {
-    font-weight: normal !important;
+    font-weight: normal;
+    display: flex;
```

Comment:
@awesomerobot this regressed the spacing on `"Unassign… @group"` label:

before/after

<img width="197" alt="Screenshot 2023-07-06 at 22 48 47" src="https://github.com/discourse/discourse-assign/assets/66961/769141d3-34aa-4550-8459-7265f27faf89"> <img width="197" alt="Screenshot 2023-07-06 at 22 51 58" src="https://github.com/discourse/discourse-assign/assets/66961/998adf4f-7eda-48f2-bbdd-7dd92b6fca98">